site/_data/fuguShowcase.js

```diff
@@ -20,25 +20,47 @@
 
 const path = require('path');
 const fs = require('fs');
+const {default: fetch} = require('node-fetch');
 
-module.exports = () => {
+async function getContentLength(url) {
+  try {
```

Comment:
This should happen when you build fugu-showcase.json file, in build-exernal. Fetching all images in _data will cause the images to be fetched for _every_ build of dcc, and this is too heavy.

Reply:
I have outsourced this now directly into https://googlechromelabs.github.io/fugu-showcase/data/data.json where I have this information available already: https://github.com/GoogleChromeLabs/fugu-showcase/commit/d98f8ad3b228e846ea1baa31d230b0a7ebf9ff41#diff-d6299b5bb2afc0956126a0adb12fd89829e01a4cc025e7277a092228b96f32e4.